metadata.rb

```diff
@@ -4,7 +4,7 @@
 license           'Apache 2.0'
 description       'Manages client.rb configuration and chef-client service'
 long_description  IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version           '4.2.4'
+version           '4.2.5'
```

Comment:
Please don't increment versions in PRs. We will change the version for releases.